src/offset_date_time.rs

```diff
@@ -546,6 +546,110 @@ impl OffsetDateTime {
     }
 }
 
+/// Methods that replace the following components `Date`, `Time`, `PrimitiveDateTime`, `Offset`.
+impl OffsetDateTime {
+    /// Replacing with a new `Time` component. The new `PrimitiveDateTime` value is stored as UTC internally.
+    ///
+    /// The return value is a `Offset_date_time`.
+    ///
+    /// ```rust
+    ///   # use time_macros::{datetime,offset,time};
```

Comment:
```suggestion
    /// # use time_macros::{datetime, offset, time};
```

Just some basic formatting to keep things tidy.